.golangci.yml

```diff
@@ -129,16 +18,72 @@ linters:
     - exhaustruct
     - forcetypeassert
     - funlen
-    - gci
     - gochecknoglobals
     - gocognit
     - goconst
     - ireturn
     - lll
     - mnd
     - nlreturn
+    - noinlineerr
     - perfsprint
-    - tenv
+    - revive
     - varnamelen
     - wrapcheck
     - wsl
+    - wsl_v5
```

Comment:
produced many errors (and we're already ignoring `wsl`)

Reply:
It is fine, thx u